misc/plunk-gen.js

```diff
@@ -84,6 +88,7 @@ function generatePlnkrContent(componentName, demoName) {
     <input type="hidden" name="description" value="Example usage of the ${componentName} widget from https://ng-bootstrap.github.io">
 ${generateTags(['Angular', 'Bootstrap', 'ng-bootstrap', capitalize(componentName)])}
     <input type="hidden" name="files[index.html]" value="${he.encode(generateIndexHtml())}">
+    <input type="hidden" name="files[css/demos.css]" value="${he.encode(generateDemosCSS())}">
```

Comment:
Is there a reason for having this file name different in plunker and stackblitz?

Reply:
The only reason is that stackblitz tries to follow cli structure. Will align plunker with it actually, thanks!